benchmarks/scala-sat/cafesat/build.sbt

```diff
@@ -1,41 +1,12 @@
-lazy val cafesat = taskKey[File]("Create the main run script")
+lazy val scalaSAT = RootProject(uri("../."))
 
-lazy val scalaSMTLib = RootProject(uri("../scala-smtlib"))
-
-lazy val runnerScriptTemplate =
-  """#!/bin/sh
-java -classpath "%s" %s "$@"
-"""
-
-exportJars := true
-
-cafesat := {
-  val cp = (fullClasspath in Runtime).value
-  val mainClass = "cafesat.Main"
-  val contents = runnerScriptTemplate.format(cp.files.absString, mainClass)
-  val out = target.value / "cafesat"
-  IO.write(out, contents)
-  out.setExecutable(true)
-  out
-}
-
-lazy val root = (project in file(""))
+lazy val scalaCafeSAT = (project in file("."))
   .settings(
     name := "CafeSat",
+    organization := "com.regblanc",
     version := "0.01",
```

Comment:
Should we bump this value to avoid conflicts with a locally cached version of `0.01`?

Reply:
Working on it. Need to add support for it into `dependency-cafesat` so that Renaissance contains a snapshot of that repository.

Reply:
That said, I'm not sure how it could conflict with a locally cached version, when we don't have it as a (maven-style) dependency, we don't publish it locally (yet), and it does not have an artifact on Maven central (the `cafesat` repo does not contain any version tag, so the version `0.01` just represents something experimental). But I think we should eventually switch to publishing dependencies locally, so I'll add some way of identifying the specific version w.r.t. our snapshot, similar to what we have in `dependency-scala-smtlib`.

Reply:
Done.